### PR TITLE
30784 dotnet   info workload info

### PR DESCRIPTION
**Pull request by JL03-Yue**
#30784 Temporary fix of adding workload info from dotnet/sdk to `dotnet --info`

### CODE REVIEW

File: src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs [hunk 1]
```diff
@@ -77,6 +83,9 @@ private static int ProcessArgs(ParseResult parseResult)
             if (parseResult.HasOption(InfoOption) && parseResult.RootSubCommandResult() == "workload")
             {
                 ShowWorkloadsInfo(parseResult);
+                IReporter reporter = null;
+                reporter ??= Cli.Utils.Reporter.Output;
+                reporter.WriteLine("");
```
Comment (dsplaisted) [inline src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs:88]: I think this does the same thing and is simpler:

```suggestion
                Cli.Utils.Reporter.Output.WriteLine("");
```

**Review by dsplaisted — APPROVED** [review]

File: src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs [hunk 2]
```diff
@@ -29,9 +29,16 @@ public static Command GetCommand()
             return Command;
         }
 
-        internal static void ShowWorkloadsInfo(ParseResult parseResult, IWorkloadInfoHelper workloadInfoHelper = null, IReporter reporter = null)
+        internal static void ShowWorkloadsInfo(ParseResult parseResult = null, IWorkloadInfoHelper workloadInfoHelper = null, IReporter reporter = null)
         {
-            workloadInfoHelper ??= new WorkloadInfoHelper(parseResult.HasOption(SharedOptions.InteractiveOption));
+            if(workloadInfoHelper != null)
+            {
+                workloadInfoHelper ??= new WorkloadInfoHelper(parseResult.HasOption(SharedOptions.InteractiveOption));
```
Comment (baronfel) [inline src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs:36]: `parseResult` here could be null, might need to guard this access with a check.

**Review by baronfel — APPROVED** [review]
:shipit:

File: src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs (hunk 2 above)
Comment (JL03-Yue) [reply src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs:36]: Does the if condition below this line work as a check?

**Pull request merged**